### PR TITLE
Menditect Overview

### PR DESCRIPTION
**Pull request by Karuna-Mendix**
https://mendix.atlassian.net/browse/TW-2371

### CODE REVIEW

File: content/en/docs/appstore/use-content/partner-solutions/mta/_index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Menditect Test Automation"
+url: /appstore/partner-solutions/mta/
+linktitle: "MTA"
+description: "Introduces Menditect Test Automation, which is new Mendix partner for automated testing."
+---
+
+## 1 Introduction
+
+[Menditect Test Automation (MTA)](https://marketplace.mendix.com/link/component/120116) is a tool for automated regression testing. MTA offers a no-code, visual suite of automated test tools for embedding unit, component, process and API testing into your application lifecycle. For more information, see the [Automated testing for Mendix] (link to be added) (mendix.com) and the [Menditect](https://menditect.com/) website.
```
Comment (Karuna-Mendix) [inline content/en/docs/appstore/use-content/partner-solutions/mta/_index.md]: waiting for the link from Marek Jednaszewski.

File: content/en/docs/appstore/use-content/partner-solutions/mta/_index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+linktitle: "MTA"
+description: "Introduces Menditect Test Automation, which is new Mendix partner for automated testing."
+---
+
+## 1 Introduction
+
+[Menditect Test Automation (MTA)](https://marketplace.mendix.com/link/component/120116) is a tool for automated regression testing. MTA offers a no-code, visual suite of automated test tools for embedding unit, component, process and API testing into your application lifecycle. For more information, see the [Automated testing for Mendix] (link to be added) (mendix.com) and the [Menditect](https://menditect.com/) website. 
+
+{{% alert color="info" %}}
+For more information about purchasing MTA, contact your Customer Support Manager (CSM) or fill out the contact [form](link to be added).
```
Comment (Karuna-Mendix) [inline content/en/docs/appstore/use-content/partner-solutions/mta/_index.md]: waiting for the form link from Veroniek Geerts

File: content/en/docs/appstore/use-content/partner-solutions/mta/_index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+* The application model being tested must be stored in the Mendix Team Server repository. 
+* The MTA application and the application under test must be able to establish network connectivity with the Mendix platform APIs, including the Mendix Team Server and Project APIs.
+
+Currently, MTA cannot be used with an on-premises Git repository or in a fully air-gapped scenario.
+
+{{% alert color="info" %}}
+MTA is offered as a paid add-on separate from the Mendix subscription. As a Mendix-based application, it requires a Mendix license and cloud resources, which must be obtained separately.
+{{% /alert %}}
+
+## 4 Read More
```
Comment (Karuna-Mendix) [inline content/en/docs/appstore/use-content/partner-solutions/mta/_index.md:34]: I am not sure if these are the good references to be added in this section. Please suggest Arjan Hendriksen

Comment (arjan-h) [reply content/en/docs/appstore/use-content/partner-solutions/mta/_index.md:34]: I would leave out the second link below.

**Comment by Luyao-Zhang-1** [comment]
Because of the merge conflicts, all the changes have been moved to https://github.com/mendix/docs/pull/8094 and we will continue to work from there.

**Comment by Karuna-Mendix** [comment]
moved changes to https://github.com/mendix/docs/pull/8094

**Pull request closed without merging**